### PR TITLE
Nisha | Test | table header Sticky in overall giddh

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86cwf7nuh

### CODE REVIEW

File: apps/web-giddh/src/app/shared/layout/layout.component.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #content-wrapper {
-    min-height: calc(100vh - 120px);
-    height: 100%;
+    height: calc(100vh - 120px);
+    // height: 100%;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/layout/layout.component.scss]:
```suggestion
```

Comment (dvCodeWorld) [reply apps/web-giddh/src/app/shared/layout/layout.component.scss]: user css class if possible

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -9454,10 +9455,10 @@ ngx-daterangepicker-material {
 #HW_badge_cont {
     visibility: visible !important;
     width: auto !important;
-    top: -5px;
+    top: 1px;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/css/style-2.scss:9458]: Why change this it part of header ?

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss:9458]: Yes, this is part of the header, but the button is not working when clicked below. Please check it on the test.

File: apps/web-giddh/src/app/shared/layout/layout.component.scss (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/shared/layout/layout.component.scss]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**